Formula/argocd-vault-plugin.rb

```diff
@@ -0,0 +1,34 @@
+class ArgocdVaultPlugin < Formula
+  desc "Argo CD plugin to retrieve secrets from Secret Management tools"
+  homepage "https://github.com/IBM/argocd-vault-plugin"
+  url "https://github.com/IBM/argocd-vault-plugin.git",
+    tag:      "v1.1.2",
+    revision: "b91d267dfcc69e8a0f029ca5afed585456f537a3"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+
+    datetime = `date -u +'%Y-%m-%dT%H:%M:%SZ'`
+
+    ldflags = %W[
+      -s -w
+      -X github.com/IBM/argocd-vault-plugin/version.Version=#{version}
+      -X github.com/IBM/argocd-vault-plugin/version.BuildDate=#{datetime}
```

Comment:
```suggestion
      -X github.com/IBM/argocd-vault-plugin/version.BuildDate=#{Time.now.utc.strftime("%Y-%m-%dT%H:%M:%SZ")}
```
Let's use this so we don't need to shell out to generate the date.